src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -130,6 +130,14 @@ public function process(ContainerBuilder $container)
 
         $dashboardGroupsSettings = $container->getParameter('sonata.admin.configuration.dashboard_groups');
         if (!empty($dashboardGroupsSettings)) {
+            uasort($dashboardGroupsSettings, static function ($a, $b) {
+                if ($a['position'] === $b['position']) {
+                    return 0;
+                }
+
+                return ($a['position'] < $b['position']) ? -1 : 1;
```

Comment:
Let's use the spaceship operator, it's its use case:
```suggestion
                return $a['position'] <=> $b['position'];
```

Reply:
Done.
Thanks for this feedback, I'm not used to using it yet.

Reply:
Yeah that's exactly right, I tried using the multiline comment feature, but it does not seem to work with suggestions yet.